be-bop-wizard/be-bop-wizard.sh

```diff
@@ -874,6 +894,28 @@ validate_bebop_config_domain() {
     return 0
 }
 
+# Build list of domains for SSL certificates and nginx configuration
+# Rules:
+#   www.domain.com → domain.com, www.domain.com, s3.domain.com
+#   domain.com → domain.com, s3.domain.com (no www)
+#   app.domain.com → app.domain.com, s3.app.domain.com (s3 prefix on subdomain)
+build_domain_list() {
```

Comment:
I find this function useful, but the name could be improved: This function builds the list of domains for nginx. Why not call it as such? (`domains_for_nginx`).

Also, the function does not need to require an argument: You can use `get_fact` to get the necessary information.